pkg/phlaredb/symdb/resolver_pprof.go

```diff
@@ -13,193 +10,114 @@ type pprofProtoSymbols struct {
 	profile googlev1.Profile
 	symbols *Symbols
 	samples *schemav1.Samples
-	tree    *model.StacktraceTree
+	lut     []uint32
```

Comment:
What does it stands for ? Locations.... ?

Reply:
Oh, that's yet another [lookup table](https://en.wikipedia.org/wiki/Lookup_table)